Shared/Scripts/BuildXLLkgVersion.cmd

```diff
@@ -1,4 +1,3 @@
-﻿REM @echo off

```

Comment:
I think the script in our release pipeline that automatically updates these LKG version files is the one that adds BOM.  Did you update those?

Reply:
Good catch, I have not

Reply:
Edgar has updated the CI scripts, so it should be good from now on.